test/src/runner/test-runner/regular-test-runner.js

```diff
@@ -305,13 +305,14 @@ describe("runner/test-runner/regular-test-runner", () => {
                 const test = new Test({});
                 const onFail = sinon.stub().named("onFail");
                 const runner = mkRunner_({ test }).on(Events.TEST_FAIL, onFail);
+                const err = new Error();
 
                 const workers = mkWorkers_();
-                workers.runTest.rejects(new Error());
+                workers.runTest.rejects(err);
 
                 await run_({ runner, workers });
 
-                assert.calledOnceWith(onFail, sinon.match(test));
+                assert.calledOnceWith(onFail, sinon.match(Object.assign(test, { err })));
```

Comment:
`onFail` as before is called with test object, containing "err" property
Now `test` is `undefined`, while before it was not defined (yeah, these things are different for sinon.match).
I.E:
`sinon.match({ foo: undefined })` and `sinon.match({ })` are different things.